test/core/util/port_isolated_runtime_environment.cc

```diff
@@ -28,18 +29,21 @@
 #include "test/core/util/port.h"
 
 #define MIN_PORT 49152
-#define MAX_PORT 65536
+#define MAX_PORT 65535
```

Comment:
Why change this?

Reply:
because the highest legal port number is 65535.